### PR TITLE
LTD-5287: Include submitted by column when displaying Archived applications

### PR DESCRIPTION
**Pull request by saruniitr**
## Change description

Designs required displaying 'submitted by' column for archived applications.
This was missed in the initial change and fixed in this PR.

This field is missing in the serializer so API changes are also required.
https://github.com/uktrade/lite-api/pull/2129

Updated tests to also verify the data rather than just the count of applications.

[LTD-5287](https://uktrade.atlassian.net/browse/LTD-5287)

[LTD-5287]: https://uktrade.atlassian.net/browse/LTD-5287?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by currycoder — APPROVED** [review]

File: exporter/templates/applications/applications.html [hunk 1]
```diff
@@ -52,6 +52,9 @@ <h1 class="govuk-heading-l">
 				<tr class="govuk-table__row">
 					<th class="govuk-table__header" scope="col">{% lcs "applications.ApplicationsSummaryPage.YOUR_REFERENCE" %}</th>
 					<th class="govuk-table__header" scope="col"></th>
+					{% if selected_filter == "archived_applications" %}
+						<th class="govuk-table__header" scope="col">Submitted by</th>
+					{% endif %}
```
Comment (currycoder) [inline exporter/templates/applications/applications.html:57]: Just thinking, should this in fact be present in all filters *except* drafts?

File: unit_tests/exporter/applications/views/test_applications_list.py [hunk 2]
```diff
@@ -8,13 +10,36 @@
 from django.urls import reverse
 from pytest_django.asserts import assertTemplateUsed
 
+from core.builtins.custom_tags import str_date
+
 faker = Faker()
```
Comment (markj0hnst0n) [inline unit_tests/exporter/applications/views/test_applications_list.py:16]: Is this faker var used anywhere?  Had a look through the file and couldn't find any references.

File: exporter/templates/applications/applications.html (hunk 1 above)
Comment (saruniitr) [reply exporter/templates/applications/applications.html:57]: that is definitely possible but designs only added in 'Archived'.
I will check with UCD and update this accordingly.

File: unit_tests/exporter/applications/views/test_applications_list.py (hunk 2 above)
Comment (saruniitr) [reply unit_tests/exporter/applications/views/test_applications_list.py:16]: it was used earlier, I will remove it now

Comment (markj0hnst0n) [reply unit_tests/exporter/applications/views/test_applications_list.py:16]: Happy enough to approve on this basis

**Review by markj0hnst0n — APPROVED** [review]

**Pull request merged**